test/stress.c

```diff
@@ -61,7 +61,7 @@ int read_from_fd()
     unsigned char c;
     int len = read(input_fd, &c, 1);
     if (len <= 0)
-	return EOF;
+    return EOF;
```

Comment:
The above is my mistake, I miss this line to change.

I use sublime text's tool to convert the tabs to 4 spaces.

In fact, the original code is a messy mixture of tabs(8 spaces) and 4 spaces. Finally, I choose 4 space because of the original indent is 4 space width.